code/game/gamemodes/game_mode.dm

```diff
@@ -53,11 +54,17 @@
 
 //pre_pre_setup() For when you really don't want certain jobs ingame.
 /datum/game_mode/proc/pre_pre_setup()
+
 	return 1
 
 ///pre_setup()
 ///Attempts to select players for special roles the mode might have.
 /datum/game_mode/proc/pre_setup()
+	cultdat = pick(all_cults)
+	to_chat(world,"[cultdat.entity_name]")
+	if(!cultdat)
+		to_chat(world, "<span class='danger'>Failed to select a cult datum, Shank a coder.</span>")

```

Comment:
I am unsure if this should be removed as you know..it tells us shit went wrong..